exoctk/atmospheric_retrievals/aws_config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "ec2_id" : "i-07717dc4bee75607a",
+    "ssh_file" : "/Users/bourque/.ssh/exoctk-kramer.pem"
+}
```

Comment:
Why is this here? Even if this is weird material pulled from another branch, this looks like something probably only Matthew can use and prob should not be committed.

Reply:
Good catch!  I changed it back to blank values.